cfme/tests/configure/test_default_views_infra.py

```diff
@@ -41,6 +42,25 @@ def set_and_test_default_view(group_name, view, page):
     assert tb.is_active(view), "{} view setting failed".format(view)
     DefaultView.set_default_view(group_name, old_default)
 
+
+def check_vm_visibility():
+    view = navigate_to(Vm, 'All')
+    value = view.sidebar.vmstemplates.tree.read_contents()
+    # Selecting the path and last Vm in the accordion to perform click
+    tree_root = value[0]
+    tree_provider = value[1][0][0]
+    tree_datacenter = value[1][0][1][0][0]
```

Comment:
When I use this index path against rhv41 provider, I get the first character of the datacenter, not the whole name.

Reply:
okay will work on your suggestion.